### DISCUSSION TITLE
Template for insurance overview

### DISCUSSION THREAD

**Discussion by zikkz** [Budibase/budibase#7577] [Ideas]
Hi all.
I've install Budibase on my Synology NAS just for testing. I'm trying to create a new template to add all my insurances. Does anyone already created a template like that and would like to share it?
Thanks in advance.

Best regards.

**Comment by melohagan** [comment]
Hi @zikkz 

Could you please clarify what you mean by 
> to add all my insurances

**Reply by zikkz** [reply]
What I mean is: I would like to have a dashboard where I can see which insurance I have with which insurer, contact of the broker, date of contract start, cancellation period, monthly/annual costs and (if possible) upload the contracts in PDF form.

**Reply by melohagan** [reply]
I get you! An insurance policy details portal
